login/views.py

```diff
@@ -0,0 +1,34 @@
+import json
+from django.shortcuts import render
+from django.http import HttpResponse, HttpResponseBadRequest
+from django.views.generic import View
+from django.contrib.auth.views import LoginView
+from django.contrib.auth import authenticate
+
+
+class IndexView(LoginView):
```

Comment:
@zairiel243  what is this used for? you have post but you return render. That is not the proper used for post methods.